packages/base/src/backbone-patch.ts

```diff
@@ -44,7 +44,9 @@ import * as utils from './utils';
 // This *MUST* be called with the model as the `this` context.
 export
 function set(key, val, options) {
-    if (key == null) return this;
```

Comment:
This is a direct patch of the backbone source, so I'd rather keep the `==` here for better compatibility.